rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -328,19 +339,24 @@ def test_upgrade_v4_v5(globaldb):
             global_dir=globaldb._data_directory / 'global_data',
             db_filename=GLOBAL_DB_FILENAME,
         )
-
     assert globaldb.get_setting_value('version', None) == 5
-
+    # insert some values in unique_cache table.
+    with globaldb.conn.write_ctx() as write_cursor:
+        # check that values are replaced and not appended
+        globaldb_set_cache_values(
+            write_cursor=write_cursor,
+            key_parts=['TEST_KEY', '123'],
+            values=['klm'],
+        )
```

Comment:
we should have already tests for this function in other places. I believe is better to move this part of the test to that file. Also I'm not sure why you are using `TEST_KEY`, that value is not something that should go to the project